temporalio/client.py

```diff
@@ -831,6 +832,108 @@ async def list_schedules(
             )
         )
 
+    async def update_worker_build_id_compatability(
```

Comment:
```suggestion
    async def update_worker_build_id_compatibility(
```

Just caught this, sorry. Update everywhere.

Reply:
Baaaaah I do that every time lol. Thanks for catching.